tests/test-cases/Group12-VCH-BC/12-01-Delete.robot

```diff
@@ -19,18 +19,17 @@ Suite Setup  Install VIC 1.1.1 to Test Server
 Test Teardown  Run Keyword If Test Failed  Clean up VIC Appliance And Local Binary
 
 *** Keywords ***
-Clean up VIC Appliance And Local Binary
-    Cleanup VIC Appliance On Test Server
-    Run  rm -rf vic.tar.gz vic
-
 Install VIC 1.1.1 to Test Server
     Log To Console  \nDownloading VIC 1.1.1 from gcp...
-    ${rc}  ${output}=  Run And Return Rc And Output  wget https://storage.googleapis.com/vic-engine-releases/vic_1.1.1.tar.gz -O vic.tar.gz
-    ${rc}  ${output}=  Run And Return Rc And Output  tar zxvf vic.tar.gz
+    ${val}=  Evaluate  time.clock()  modules=time
```

Comment:
Seems as though this should be using `Install VIC with version to Test Server` but maybe there's a reason? Again, not something that needs addressing in this PR.